installing/disconnected_install/installing-mirroring-disconnected.adoc

```diff
@@ -0,0 +1,133 @@
+:_content-type: ASSEMBLY
+[id="installing-mirroring-disconnected"]
+= Mirroring images for a disconnected installation using the oc-mirror plug-in
+include::_attributes/common-attributes.adoc[]
+:context: installing-mirroring-disconnected
+
+toc::[]
+
+////
+General to do list:
+* Confirm that no new content was added since I did the reorg of the disconnected install assemblies (up to date as of 2/13)
+* Add info about mirroring to a namespace
+* Test and get some actual example output
+* Use cases for changing image set config between runs
+////
+
+You can use the procedures in this section to ensure your clusters only use container images that satisfy your organizational controls on external content. Before you install a cluster on infrastructure that you provision in a restricted network, you must mirror the required container images into that environment. To mirror container images, you must have a registry for mirroring.
+
+You can use the oc-mirror OpenShift CLI (`oc`) plug-in to mirror images to a mirror registry in your fully or partially disconnected environments.
+
+:FeatureName: Mirroring images for disconnected environments using the oc-mirror plug-in
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+The following steps outline the high-level workflow on how to use the oc-mirror plug-in to mirror images to a mirror registry:
+
+. Create an image set configuration file.
+. Mirror the image set to the mirror registry.
+. Installing the `ImageContentSourcePolicy` and `CatalogSource` resources that were generated by oc-mirror into the cluster.
+. Repeat these steps to update your mirror registry as necessary.
+
+// About the oc-mirror plug-in
+include::modules/oc-mirror-about.adoc[leveloffset=+1]
+
+// TODO: Check the content of this module to make sure it's accurate for oc-mirror
+include::modules/installation-about-mirror-registry.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For information about viewing the CRI-O logs to view the image source, see xref:../../installing/validating-an-installation.adoc#viewing-the-image-pull-source_validating-an-installation[Viewing the image pull source].
+
+// TODO: Copied this prereq section + about mirror registry from the orig section. Review/update as necessary
+[id="prerequisites_installing-mirroring-disconnected"]
+== Prerequisites
```

Comment:
I see what you mean. At first, the placement made sense to me since it follows after two "About" modules, so that you learn about the oc-mirror tool and about the mirror registry. Though this might be a chicken and egg thing, where why do you need to learn about the mirror registry til you know it's a prereq? 

Honestly though, the only prereq listed is the mirror registry. I might move the prereqs above the mirror registry module for now, but I'll re-evaluate it for the GA release to see if they should be combined, so it's not so redundant.